velox/functions/prestosql/aggregates/MinMaxByAggregates.cpp

```diff
@@ -470,7 +470,7 @@ class MaxByAggregate : public MinMaxByAggregate<T, U> {
       typename AccumulatorTypeTraits<U>::AccumulatorType;
 
   explicit MaxByAggregate(TypePtr resultType)
-      : MinMaxByAggregate<T, U>(resultType, std::numeric_limits<U>::min()) {}
+      : MinMaxByAggregate<T, U>(resultType, std::numeric_limits<U>::lowest()) {}
```

Comment:
Good catch. Could you add to cover this in unit test? Thanks!

Reply:
Adapted the unit tests to cover negative doubles based on your suggestion :)